.github/ISSUE_TEMPLATE/AI-Assisted-Development-Story-Template.md

```
@@ -0,0 +1,73 @@
+---
+name: AI-Assisted-Development-Story Template
+about: Template for describing a new feature, enhancement, or requirement
+title: ''
+labels: ''
+assignees: ''
+---
+
+## 1. Problem Statement (Mandatory)
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)